### ISSUE TITLE
Jaccard index between same type of ground truth and segmented results

### ISSUE THREAD

**Issue by elboyran** [DynaSlum/SateliteImaging#42]
The segmentation result could be either pixel or grid Binary Mask (BM) or a (multipolygonal) Shape (S).
The ground truth can be either a shape file or sometimes - pixel mask. In this issue consider comparison of the same type: BM <-> BM and S <-> S.

Use skikit-learn Python implementation of the [Jaccard similarity score](http://scikit-learn.org/stable/modules/generated/sklearn.metrics.jaccard_similarity_score.html).

- [x] Install [scikit-learn](http://scikit-learn.org/stable/install.html)

- [x] Test simple example example documentation.

- [x] Source notebook for comparison of label masks.

- [x] Function for comparison of label masks.

- [x] Demo notebook using the function

- [x] Source notebook- toy example Jaccard index mulipolygons

- [x] Demo notebook- toy example Jaccard index multipolygons

- [x] Source notebook for comparison of shape files. Fix corrupt shape file.

- [x] Function  for shape files.

- [x] Demo notebook using the function for shape files
